### PR TITLE
Input

### PR DESCRIPTION
**Pull request by NagaSatyaGeeth**
https://8y13k.csb.app/
input

### CODE REVIEW

**Review by Divya70 — APPROVED** [review]
looks good

File: input/input.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800&display=swap");
+:root {
+	font-family: "Montserrat", sans-serif;
+	padding: 0px;
+	margin: 0px;
```
Comment (ruchiragoswami) [inline input/input.css:5]: My personal thought, I don't use units (px or rem for example) when the value is "0" for margin and padding :) 
It works!

File: input/input.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Speed CL | input</title>
+		<link rel="stylesheet" href="input.css" />
+		<link
+			href="https://fonts.googleapis.com/icon?family=Material+Icons"
+			rel="stylesheet"
```
Comment (ruchiragoswami) [inline input/input.html:11]: Just drawing your attention, this link is spread over a few lines.

File: input/input.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		/>
+		<link rel="icon" type="image/x-icon" href="speed_logo-cropped.png" />
+	</head>
+	<bo>
```
Comment (ruchiragoswami) [inline input/input.html]: bo ?
I guess it was "body"

**Review by ruchiragoswami — COMMENTED** [review]
Great work Naga. I left few comments. All looks good except for one. Please take a look.

**Review by ruchiragoswami — APPROVED** [review]
Looks good!

File: input/input.html (hunk 2 above)
Comment (NagaSatyaGeeth) [reply input/input.html:11]: can't do anything about that since i use prettify

**Review by SubhamChoudhury — APPROVED** [review]

File: input/input.html (hunk 3 above)
Comment (NagaSatyaGeeth) [reply input/input.html]: yeah fixed this

**Pull request merged**